solution/app/src/main/java/com/google/codelabs/maps/placesdemo/CurrentPlaceActivity.kt

```diff
@@ -42,17 +51,21 @@ import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.launch
 
 @ExperimentalCoroutinesApi
-class CurrentPlaceActivity : AppCompatActivity() {
+class CurrentPlaceActivity : AppCompatActivity(), OnMapReadyCallback {
     private lateinit var placesClient: PlacesClient
     private lateinit var currentButton: Button
     private lateinit var responseView: TextView
+    private var map: GoogleMap? = null
+    private val defaultLocation = LatLng(-33.8523341, 151.2106085)
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_current)
 
         // Retrieve a PlacesClient (previously initialized - see DemoApplication)
```

Comment:
We should demonstrate initializing the PlacesClient using PLACES_API_KEY per https://developers.google.com/maps/documentation/places/android-sdk/config#connect-client